config/owasp/suppressions.xml

```diff
@@ -44,6 +44,16 @@
     <gav regex="true">^org\.springframework\.security:.*$</gav>
     <cve>CVE-2018-1258</cve>
   </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+        file name: applicationinsights-core-2.3.1.jar (shaded: io.netty:netty-tcnative-boringssl-static:2.0.17.Final)
+      ]]>
+    </notes>
+    <gav regex="true">^io\.netty:netty-tcnative-boringssl-static:.*$</gav>
+    <cve>CVE-2015-2156</cve>
+    <cve>CVE-2014-3488</cve>
+  </suppress>
   <!-- Temporary suppression until dependencies are upgraded -->
   <suppress>
     <notes><![CDATA[file name: spring-security-rsa-1.0.7.RELEASE.jar]]></notes>
```

Comment:
Upgrade to latest known version does not solve the problem. Other teams have suppressed it as well. Is looks like it has to stay there for a moment.